Improve B1 shimming: implement resampling of masks into B1 maps + phase only shimming and add documentation (#321)
shimming-toolbox/shimming-toolbox: README.rst

* Masking tools for the brain and spinal cord

> * Supports the creation and usage of a variety of coil profiles: Spherical harmonics, shim-only arrays, "AC-DC" multi-coil, etc.

Reviewer: ```suggestion
* Create and use a variety of B0 shimming coil profiles: Spherical harmonics, shim-only arrays, "AC-DC" multi-coil, etc.
```

> * Supports different shimming scenarios: dynamic (slicewise), realtime (shim modulation with respiration), gradient z-shimming, two-region shimming (e.g., fat and brain)

Reviewer: ```suggestion
* Supports different B0 shimming scenarios: dynamic (slicewise), realtime (shim modulation with respiration), gradient z-shimming, two-region shimming (e.g., fat and brain)
```

> * Supports static B1+ shimming scenarios

Reviewer: ```suggestion
* RF shimming for parallel transmit systems (a.k.a. B1+ shimming)
```
